src/clipboard_handlers/image_clipboard.ts

```diff
@@ -55,33 +55,26 @@ export class ImageClipboardHandler extends AbstractFigureClipboardHandler<Clipbo
     }
     const { zones, figureId } = target;
     const sheetId = this.getters.getActiveSheetId();
-    const numCols = this.getters.getNumberCols(sheetId);
-    const numRows = this.getters.getNumberRows(sheetId);
-    const targetX = this.getters.getColDimensions(sheetId, zones[0].left).start;
-    const targetY = this.getters.getRowDimensions(sheetId, zones[0].top).start;
-    const maxX = this.getters.getColDimensions(sheetId, numCols - 1).end;
-    const maxY = this.getters.getRowDimensions(sheetId, numRows - 1).end;
     const { width, height } = clippedContent.copiedFigure;
-    const position = {
-      x: maxX < width ? 0 : Math.min(targetX, maxX - width),
-      y: maxY < height ? 0 : Math.min(targetY, maxY - height),
-    };
+    const anchor = { col: zones[0].left, row: zones[0].top };
+    const offset = { x: 0, y: 0 };
     const copy = deepCopy(clippedContent.copiedImage);
     this.dispatch("CREATE_IMAGE", {
-      figureId,
+      figureId: figureId,
       sheetId,
-      position,
+      anchor,
+      offset,
       size: { height, width },
       definition: copy,
     });
 
     if (options.isCutOperation) {
       this.dispatch("DELETE_FIGURE", {
         sheetId: clippedContent.sheetId,
-        id: clippedContent.copiedFigure.id,
+        figureId: clippedContent.copiedFigure.id,
       });
     }
-    this.dispatch("SELECT_FIGURE", { id: figureId });
+    this.dispatch("SELECT_FIGURE", { figureId: figureId });
```

Comment:
```suggestion
    this.dispatch("SELECT_FIGURE", { figureId });
```
there's probably other places in the code with this